martin-mbtiles/src/mbtiles.rs

```diff
@@ -27,10 +30,12 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "cli", derive(ValueEnum))]
 pub enum MbtType {
-    TileTables,
-    DeDuplicated,
+    Flat,
+    FlatHashed,
```

Comment:
Name is a bit confusing I think, how about `FlatWithHash`?  Usually `hashed` anything means it is obfuscated, so lets not used that.  (i.e. instead of the password, a hash value of that password is stored, and it is not possible to recover the original password from the hash)